airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -63,23 +64,76 @@ public class TemporalClient {
    */
   private static final int DELAY_BETWEEN_QUERY_MS = 10;
 
-  private static final int MAXIMUM_SEARCH_PAGE_SIZE = 50;
+  public static TemporalClient production(final Configs configs) {
+    if (configs.temporalCloudEnabled()) {
+      log.info("TemporalClient.production chose Cloud...");
+      return TemporalClient.cloud(configs);
+    }
+    log.info("TemporalClient.production chose Airbyte...");
+    return TemporalClient.airbyte(configs);
+  }
+
+  public static WorkerFactory productionWorkerFactory(final Configs configs) {
```

Comment:
`production` makes me think we have a `non-production` instance - is that right? can we simplify to `getWorkerFactor`?